src/test/scala/dpla/ingestion3/reports/PropertyValueReportTest.scala

```diff
@@ -0,0 +1,17 @@
+package dpla.ingestion3.reports
+
+import org.scalatest.FlatSpec
+import org.scalamock.scalatest.MockFactory
+import org.apache.spark.sql.{DataFrame, Row, SparkSession, Dataset}
+import dpla.ingestion3.model._
+import dpla.ingestion3.reports._
+
+
+class PropertyValueReportTest extends FlatSpec with MockFactory{
+
+  "PropertyValueReport.process" should "x" in {
+    // FIXME: How do you mock Dataframes and DataSets??
```

Comment:
I don't think we'd not get that with embedded Spark. If we don't, however, I'd argue for moving this to an integration test over mocking Spark.